api/src/tests/account_abstraction_test.rs

```diff
@@ -15,14 +15,29 @@ use aptos_types::{
 };
 use move_core_types::{identifier::Identifier, language_storage::ModuleId, vm_status::StatusCode};
 use rand::rngs::OsRng;
+use rstest::rstest;
 use serde_json::json;
 use std::{path::PathBuf, sync::Arc};
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-async fn test_account_abstraction_single_signer() {
+#[rstest(
```

Comment:
Seems reasonable. We'll all have to learn how rstest works but I do see the benefit.